lib/fog/aliyun/requests/storage/get_object.rb

```diff
@@ -11,30 +11,17 @@ class Real
         #
         def get_object(object, range = nil, options = {})
           options = options.reject { |_key, value| value.nil? }
-          bucket = options[:bucket]
-          bucket ||= @aliyun_oss_bucket
-          endpoint = options[:endpoint]
-          if endpoint.nil?
-            location = get_bucket_location(bucket)
+          bucket_name = options[:bucket]
+          bucket_name ||= @aliyun_oss_bucket
+          # Using OSS ruby SDK to fix performance issue
+          bucket = @oss_client.get_bucket(bucket_name)
+          body = Array.new
+          obj = bucket.get_object(object) do |chunk|
+            body << chunk
           end
-          resource = bucket + '/' + object
-          para = {
-            expects: [200, 206, 404],
```

Comment:
How aliyun sdk process 206 and 404 error, what will happen in case of such error? Exception thrown?

Reply:
If the http code is more than 300, there will throw an exception and the code refer to: https://github.com/aliyun/aliyun-oss-ruby-sdk/blob/master/lib/aliyun/oss/http.rb#L271

Reply:
HI @ninja-at-work It has been fixed by https://github.com/fog/fog-aliyun/pull/86/files#diff-718e497522d24547e3c39c54eb956537R134. Please check it.